app/models/loan_health_check.rb

```diff
@@ -0,0 +1,84 @@
+# == Schema Information
+#
+# Table name: loan_health_checks
+#
+#  created_at           :datetime         not null
+#  has_late_steps       :boolean
+#  has_sporadic_updates :boolean
+#  id                   :integer          not null, primary key
+#  last_log_date        :date
+#  missing_contract     :boolean
+#  progress_pct         :decimal(, )
+#  project_id           :integer
+#  updated_at           :datetime         not null
+#
+# Indexes
+#
+#  index_loan_health_checks_on_project_id  (project_id)
+#
+# Foreign Keys
+#
+#  fk_rails_6441c92912  (project_id => projects.id)
+#
+
+class LoanHealthCheck < ActiveRecord::Base
+  belongs_to :loan, class_name: Project, foreign_key: :project_id
```

Comment:
This seems odd. If it's just for loans, can't you use class_name Loan and foreign_key loan_id?